backend/src/apiserver/server/report_server_test.go

```diff
@@ -79,156 +91,156 @@ func TestReportWorkflow_ValidationFailed(t *testing.T) {
-	assert.Equal(t, err.(*util.UserError).ExternalStatusCode(), codes.InvalidArgument)
-
-	// UID
-	workflow = util.NewWorkflow(&v1alpha1.Workflow{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "Workflow",
-			APIVersion: "argoproj.io/v1alpha1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "MY_NAME",
-			Namespace: "MY_NAMESPACE",
-			OwnerReferences: []metav1.OwnerReference{{
-				APIVersion: "kubeflow.org/v1beta1",
-				Kind:       "ScheduledWorkflow",
-				Name:       "SCHEDULE_NAME",
-				UID:        types.UID("1"),
-			}},
-		},
-	})
-
-	_, err = ValidateReportWorkflowRequest(&api.ReportWorkflowRequest{Workflow: workflow.ToStringForStore()})
-	assert.NotNil(t, err)
-	assert.Contains(t, err.(*util.UserError).ExternalMessage(), "The workflow must have a UID")
-	assert.Equal(t, err.(*util.UserError).ExternalStatusCode(), codes.InvalidArgument)
-}
-
-func TestValidateReportScheduledWorkflowRequest_UnmarshalError(t *testing.T) {
-	_, err := ValidateReportScheduledWorkflowRequest(
-		&api.ReportScheduledWorkflowRequest{ScheduledWorkflow: "WRONG_SCHEDULED_WORKFLOW"})
-	assert.NotNil(t, err)
-	assert.Equal(t, codes.InvalidArgument, err.(*util.UserError).ExternalStatusCode())
-	assert.Contains(t, err.Error(), "Could not unmarshal")
-}
-
-func TestValidateReportScheduledWorkflowRequest_MissingField(t *testing.T) {
-	// Name
-	swf := util.NewScheduledWorkflow(&swfapi.ScheduledWorkflow{
-		ObjectMeta: metav1.ObjectMeta{
-			Namespace: "MY_NAMESPACE",
-			UID:       "1",
-		},
-	})
-
-	_, err := ValidateReportScheduledWorkflowRequest(
-		&api.ReportScheduledWorkflowRequest{ScheduledWorkflow: swf.ToStringForStore()})
-	assert.NotNil(t, err)
-	assert.Contains(t, err.(*util.UserError).ExternalMessage(), "The resource must have a name")
-	assert.Equal(t, err.(*util.UserError).ExternalStatusCode(), codes.InvalidArgument)
-
-	// Namespace
-	swf = util.NewScheduledWorkflow(&swfapi.ScheduledWorkflow{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "MY_NAME",
-			UID:  "1",
-		},
-	})
-
-	_, err = ValidateReportScheduledWorkflowRequest(
-		&api.ReportScheduledWorkflowRequest{ScheduledWorkflow: swf.ToStringForStore()})
-	assert.NotNil(t, err)
-	assert.Contains(t, err.(*util.UserError).ExternalMessage(), "The resource must have a namespace")
-	assert.Equal(t, err.(*util.UserError).ExternalStatusCode(), codes.InvalidArgument)
-
-	// UID
-	swf = util.NewScheduledWorkflow(&swfapi.ScheduledWorkflow{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "MY_NAME",
-			Namespace: "MY_NAMESPACE",
-		},
-	})
-
-	_, err = ValidateReportScheduledWorkflowRequest(
-		&api.ReportScheduledWorkflowRequest{ScheduledWorkflow: swf.ToStringForStore()})
-	assert.NotNil(t, err)
-	assert.Contains(t, err.(*util.UserError).ExternalMessage(), "The resource must have a UID")
-	assert.Equal(t, err.(*util.UserError).ExternalStatusCode(), codes.InvalidArgument)
-}
+// func TestValidateReportWorkflowRequest(t *testing.T) {
```

Comment:
Same here, you probably need to uncomment these tests and make sure they all pass.

Reply:
This test is no longer needed as the function has been removed. Same applied to other commented functions. I will remove them in the upcoming commits and add unit-tests for v2beta1 functions and methods. I kept them as I may recycle some of their content.

Reply:
remove?

Reply:
(reopen)

Reply:
(forgot to push?) @gkcalat

Reply:
Haven't pushed yet. I will ping you when it's ready for review